ammico/summary.py

```diff
@@ -2,34 +2,66 @@
 from torch import cuda, no_grad
 from PIL import Image
 from lavis.models import load_model_and_preprocess
+from typing import Optional
 
 
 class SummaryDetector(AnalysisMethod):
+    allowed_model_types = [
+        "base",
+        "large",
+    ]
+    allowed_new_model_types = [
```

Comment:
why do we distinguish the two `allowed_model_types` and `allowed_new_model_types`? Can we make this one list?